artemis-core-client/src/main/java/org/apache/activemq/artemis/api/config/ActiveMQDefaultConfiguration.java

```diff
@@ -477,6 +477,16 @@ public static String getDefaultHapolicyBackupStrategy() {
 
    public static int DEFAULT_QUORUM_SIZE = -1;
 
+   public static final boolean DEFAULT_ANALYZE_CRITICAL = true;
+
+   public static final long DEFAULT_ANALYZE_CRITICAL_TIMEOUT = 120000;
+
+   // this will be 0, the implementation should return 1/2 of the configured critical timeout
+   public static final long DEFAULT_ANALYZE_CHECK_PERIOD = 0;
```

Comment:
@clebertsuconic i put some other comments, maybe keep it and not have magic numbers in the other code, I've pointed out.